tools/lsp/preview/eval.rs

```diff
@@ -621,7 +608,7 @@ fn handle_builtin_function(
             impl i_slint_core::translations::FormatArgs for StringModelWrapper {
                 type Output<'a> = slint::SharedString;
                 fn from_index(&self, index: usize) -> Option<slint::SharedString> {
-                    self.0.row_data(index).map(|x| x.try_into().unwrap())
+                    self.0.row_data(index).map(|x| x.try_into().unwrap_or_default())
```

Comment:
There is a bunch of other instance of `try_into().unwrap()` in this function.  I believe they all should actually be `unwrap_or_default`

The actual evaluator can unwrap because it is meant to be precise. But your evaluator here probably has a lot of Value::Void all over and all these unwrap would panic and crash the LSP.